portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,18 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I love trying new things.', 'My favorite color is red!', 'My favorite place is Yosemite National Park.',
+       'I would be an environmental engineer if I was not a CS major!', 'I love the ocean!', 'I love traveling!',
+       'I practice Yoga.', 'I started meditating recently.', 'I love organizing!', 'I love nature!', 
+       'I would be absolutely lost without my planner.', 'Cleaning is my favorite stress reliever!', 
+       'I love adventures!', 'I am fascinated by AI.', 'I am not a vegetarian, but I rarely eat meat!'];
```

Comment:
Might be good to keep the line length within 80 characters:
https://google.github.io/styleguide/jsguide.html#formatting-column-limit

Reply:
Will do, thank you!